### PR TITLE
add new file

### LINKED ISSUE #8: Create a new file for monasethi Redlands, CA

### PR DESCRIPTION
**Pull request by monasethi**
closes #8

### CODE REVIEW

File: monasethi-redlands.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# test, ca
```
Comment (ghost) [inline monasethi-redlands.md:1]: Where is test?

**Comment by ghost** [comment]
:dromedary_camel:

File: monasethi-redlands.md (hunk 1 above)
Comment (monasethi) [reply monasethi-redlands.md:1]: its in nowhereland

**Review by ghost — APPROVED** [review]
Ok

**Pull request merged**